semantic-web-technology-introduction.markdown

```diff
@@ -57,27 +56,27 @@ Proof provides trace or explanation of the steps of logical reasoning and needs
 {% include image.html type="small-figure" url="/assets/images/W3C-languages-Venn.png" description="Figure 3: Venn diagram showing the relations between W3C languages and their syntactical features" %}
 
 Note: there is a [W3C N3 community group](https://www.w3.org/community/n3-dev/participants) to further develop N3 and bring it to a possible standard. Members are for example Tim Berners-Lee, Jos De Roo (the developer of the EYE reasoner), and Hans Cools (the author). The development is in the [GitHub W3C N3 repository](https://github.com/w3c/N3).
 
-## A reflection
+## Some reflections
 RDF is more than 'yet another data format'. Implementing the SW standards or *RDF-izing* data (as *pars pro toto*) goes way beyond long-term data storage. Especially the effort to create and/or reuse necessary ontologies, providing a sound semantic space to express source data in a formal way, is too substantial. In this sense RDF departs from quick, ad hoc, standalone development, but aims at a fundamentally different data processing of which the data translation into RDF is only the beginning. Of course development complexity and time depend on the needed semantics to express data and solve academic, industrial or other data flow and research issues. On one hand an existing ontology can suffice and a limited set of rules can solve the issues. On the other hand a library of new ontologies (mostly based on existing ones) has to be created and a series of rules sets has to be developed providing very different functionalities to solve all kinds of issues, as is the case in our environment.  
 Modeling ontologies providing the required domain semantics is not a common IT development activity. It can be pretty time-consuming, needing both width and depth. One can learn it by adopting certain best practices and efficiency, but different aspects need to be covered: semiotics, linguistics, formal logic, math, analysis, an eye for detail but also keeping an overview, and participation in conceptual discussions with domain specialists. Modeling is very iterative, likely taking a while until satisfying.  
-There are people in the RDF community who want SWT to be more digestible for 'the average IT developer'. This should not be translated into ad hoc and standalone development or too narrow interpretation of the W3C standards, deviating from the original goals of the Semantic Web. Semantic interoperability and machine reasoning should at all times be preserved.
+#There are people in the RDF community who want SWT to be more digestible for 'the average IT developer'. This should not be translated into ad hoc and standalone development or too narrow interpretation of the W3C standards, deviating from the original goals of the Semantic Web. Semantic interoperability and machine reasoning should at all times be preserved.#
```

Comment:
Would omit this part and write something that is more related to the previous section and summarizes the whole idea of SWT. Maybe like this:
_But we believe that the investment of time is worth it, because..._

Reply:
I'll have a look.